var/spack/repos/builtin/packages/eic-opticks/package.py

```diff
@@ -0,0 +1,40 @@
+from spack.package import *
+
+
+class EicOpticks(CMakePackage, CudaPackage):
+    """EIC Opticks package for GPU simulation"""
+
+    homepage = "https://github.com/bnlnpps/eic-opticks"
+    url      = "https://github.com/bnlnpps/eic-opticks/archive/tags/1.0.0-rc1.tar.gz"
+    git      = "https://github.com/bnlnpps/eic-opticks.git"
+
+    maintainers = ["plexoos"]
+
+    version("1.0.0-rc1", sha256="6c6d13d996bde8e0658f2804a9fe6ea17bb15ca0d40707d64e2afff70da4d294")
+    version("esi", branch="esi")
+
+    depends_on("cmake@3.10:", type="build")
+    depends_on("geant4")
+    depends_on("glew")
+    depends_on("glfw")
+    depends_on("glm")
+    depends_on("glu")
+    depends_on("nlohmann-json")
+    depends_on("mesa~llvm")
+    depends_on("plog")
+
+    # External resource (e.g., an additional dataset, model, or patch file)
+    resource(
```

Comment:
This resource should be installed as a separate spack package.

Reply:
https://github.com/spack/spack/pull/49990#discussion_r2037385887